packages/theme-patternfly-org/components/example/exampleToolbar.js

```diff
@@ -56,6 +58,14 @@ export const ExampleToolbar = ({
   const fullscreenLabel = 'Open example in new window';
   const convertLabel = 'Convert example from Typescript to JavaScript';
   const undoAllLabel = 'Undo all changes';
+
+  const copyAriaLabel = `Copy ${exampleTitle} examle code to clipboard`;
+  const languageAriaLabel = `Toggle ${lang.toUpperCase()} code in ${exampleTitle} examle`;
+  const codesandboxAriaLabel = `Open ${exampleTitle} example in CodeSandbox`;
+  const fullscreenAriaLabel = `Open ${exampleTitle} example in new window`;
+  const convertAriaLabel = `Convert ${exampleTitle} example from Typescript to JavaScript`;
```

Comment:
This is created but never used, it can be implemented on line 156 below `aria-label={convertLabel}`